### ISSUE TITLE
Forest Fire Detection and Prevention System

### ISSUE THREAD

**Issue by nelsonic** [mosteirodelandim/hq#3]
please ignore this placeholder issue. 

A fire in the Landim forest would _rapidly_ reach the main house which is mostly wood (inside).

[https://www.google.com/maps/place/Mosteiro+de+Santa+Maria+de+Landim/@41.3784704,-8.4626658,618m/](https://www.google.com/maps/place/Mosteiro+de+Santa+Maria+de+Landim/@41.3784704,-8.4626658,618m/data=!3m1!1e3!4m12!1m6!3m5!1s0xd24f61285cbd035:0xf11ff2ba637bd6f1!2sMosteiro+de+Santa+Maria+de+Landim!8m2!3d41.3791331!4d-8.4641372!3m4!1s0xd24f61285cbd035:0xf11ff2ba637bd6f1!8m2!3d41.3791331!4d-8.4641372!5m1!1e4)

![image](https://user-images.githubusercontent.com/194400/179374246-ca137094-1c31-4973-bf4b-0cb4ebdf6cc2.png)

# Proposed mini-project: 

Use an IBC: https://en.wikipedia.org/wiki/Intermediate_bulk_container
With some basic electronics. 
https://create.arduino.cc/projecthub/csw/flame-sensor-with-arduino-6df6b9
To create a fire detection and extinguishing system with 1k litres of water.

IBCs can be strategically placed on cinder blocks so they are slightly elevated and easily accessible
at regular intervals e.g. **`50 m`** around the top of the "cerca"

**`38 €`** on OLX (negotiable) and can be delivered to the house.
https://www.olx.pt/d/anuncio/depsitos-cubas-ibcs-1000-litros-IDFGS7I.html
![image](https://user-images.githubusercontent.com/194400/179374267-b02c23c6-c48a-4036-a413-f1ffa32cc45a.png)

# Target Price including IBC + Electronics: `€100/unit`

If we could keep the cost of the electronics below **`€50`**, 
we could have a total installed [material] costs of **`€100`**. 
And by open-sourcing the design/schematics and hosting it and a video on the **`new`** domain mosteirodelandim/landim-website#34 
we could make something _useful_ that _many_ other people could put on their properties to help prevent/extinguish fire.

**Comment by LuchoTurtle** [comment]
That sounds like an awesome idea 😍 However, we should have at least 10 of these to cover a significant area of the "cerca" and I think think the current budget allows for such a thing 🤔 . It wouldn't hurt to ask though

**Comment by nelsonic** [comment]
We don’t have to request budget to build a prototype. And the prototype can be small scale e.g just the electronics, code and water in a bucket that tests the principals.
Guarantee that if we build this and fully open source/design it, _many_ people will want one.

**Comment by nelsonic** [comment]
I'm still _super-duper_ keen to work on this as a side-project/quest. 💡 ❤️ 
@LuchoTurtle want to lead the project ad dev the spec so we can build a prototype? 🏗️ 
Would be _amaze_ to build low-cost/energy autonomous forest fire protection systems 🧯 
that we can Open Source for _anyone_ to use. 💭

**Comment by nelsonic** [comment]
Let's start by asking Ricardo if there are any "spare" IBCs in Landim that we can use.
If not, then we need to buy one on `OLX` and get it delivered to site so can explore the options.

**Comment by LuchoTurtle** [comment]
I don't think we actually need an IBC to test this as a prototype. I don't know yet how this will actually be implemented with an IBC (a little door opens in an IBC once a flame is detected? A pump is installed that sprays water once the flame is detected?)

I'm thinking about how this will be placed around the woods. Given that the woods have a certain degree of incline, it makes sense to place them above so the water flows downstream.

However:
- if we're using Arduino, how long will the battery last?
- there are [flame sensors](https://www.google.com/search?q=Flame+Sensor&sourceid=chrome&ie=UTF-8), for sure. But what about false positives? If there's no flame, we've just wasted 1000L of water.
- if one IBC is pumped, we must know where and when. This will mean we need internet access to get notifications. I don't think there's any service in the woods.

Tio Tó has a system in his vineyard that uploads temp and moisture levels in 5 seconds intervals to his server. I might ask him how he manages internet access and the battery life of the sensors out in the field.

**Comment by nelsonic** [comment]
@LuchoTurtle very good idea to get Tio Tó's input/help as he has lots of experience. 💡 
However, I'd caution you to avoid _framing_ the discussion in terms of an electrical engineering project. ⚡ 
I think we should try and build something with **_zero_ `electronics`**. 🙅 
Or at least not for critical systems i.e: we need a heat-based trigger 🔫

and a gravity based low-pressure sprinkler 🚰 
so that any **`electronics`** we implement for _monitoring_ can be **100% optional**. 🔌 
This keeps costs low and reliability high. 💭 

One option could be to use "Flash String" 

See: https://www.propdog.co.uk/flash-products/flash-string

<img width="1310" alt="image" src="https://user-images.githubusercontent.com/194400/228841013-defbcb13-b0eb-4eb0-b7d6-16688ebc12b0.png">

It's _obvs_ not "cheap" per **`meter`** but we would only need **`~1M`** per deployment so **`~€2.20`**.
If the string ignites because there is a flame the sprinkler will be activated. 

Watch: https://youtu.be/x4j2ZRTqNT0 for a quick demo
<img width="1031" alt="image" src="https://user-images.githubusercontent.com/194400/228840484-808a09a9-7024-4e40-b360-0bd692767313.png">

The string doesn't need a lot of tensile strength, just enough to hold a ball valve closed. 
Then of course we can have `IoT` monitoring systems in place for early detection
but as a general application with mass appeal the system should be free of electronics. 💭

**Comment by LuchoTurtle** [comment]
I had no idea things like "Flash strings" existed, it looks mighty useful. What was your idea on using this? Use it to open up the "gate" once the string gets burnt?

**Comment by nelsonic** [comment]
Yeah, have a handle that has a spring attached to pull it open and have the Flash String holding it closed. 
If the string burns the water valve is opened triggering a sprinkler that puts out the fire.
The real trick in all this is figuring out what sort of Radius a sprinkler could have.
There wouldn't be much point if it could only reach 2m around the IBC ... 💭 

Another zero electronics (and therefore cheap) option 
could be to use something akin to a ziplock bag full of water attached to a tree
if the bag burns the water is released extinguishing the fire. 
ZipLock so that the water doesn't simply evaporate before there's a fire!

https://www.alibaba.com/product-detail/China-Supply-Wholesale-Durable-5-Mil_1600827101407.html?spm=a2700.galleryofferlist.normal_offer.d_title.340aea47eXZ3Q5
![image](https://github.com/mosteirodelandim/hq/assets/194400/b88d3b07-ee3c-4adc-bd41-86b22037f84c)

For reference if we ordered 3k of these 2-gallon ziplock bags it's only 180 euros. 

![image](https://github.com/mosteirodelandim/hq/assets/194400/173a9e19-a59f-42af-be83-48b3158e99b0)

Then we could spend an afternoon nailing a few of them to the base of each tree to be protected. 
The _obvs_ downside of this approach is the eyesore of having bags around trees in an otherwise pristine forest. 🙄 
It's a balancing act. ⚖️ 
If there's a fire the response time might be too slow to save the forest. ⏳ 
Eucalyptus are known to be highly flammable https://www.quora.com/Do-eucalyptus-trees-catch-fire-easily 🌲 🔥

**Comment by nelsonic** [comment]
https://www.bbc.com/news/business-66266186
